zebedee-cms/src/main/java/com/github/onsdigital/zebedee/json/publishing/PublishedCollection.java

```diff
@@ -2,12 +2,19 @@
 
 import com.github.onsdigital.zebedee.json.CollectionBase;
 import com.github.onsdigital.zebedee.json.CollectionType;
+import com.github.onsdigital.zebedee.json.CollectionDataset;
+import com.github.onsdigital.zebedee.json.CollectionDatasetVersion;
 
 import java.util.Date;
 import java.util.List;
+import java.util.Set;
+
 
 public class PublishedCollection extends CollectionBase {
 
+    public Set<CollectionDataset> datasets;
```

Comment:
Java convention is that these should be private and there should be accessor methods. Other than than all good.